components/edit-collective/EditCollectiveUpdates.js

```diff
@@ -0,0 +1,118 @@
+import { Flex } from '@rebass/grid';
+import { set, cloneDeep, pick } from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useMutation } from '@apollo/react-hooks';
+import { useIntl, defineMessages } from 'react-intl';
+import styled from 'styled-components';
+
+import imgPreviewList from '../../public/static/images/updates/updates-list-preview.png';
+import imgPreviewNewUpdate from '../../public/static/images/updates/updates-new-preview.png';
+
+import { getErrorFromGraphqlException } from '../../lib/errors';
+import hasFeature, { FEATURES, FEATURE_FLAGS } from '../../lib/allowed-features';
+import CreateUpdateFAQ from '../faqs/CreateUpdateFAQ';
+import Container from '../Container';
+import MessageBox from '../MessageBox';
+import { H3, P } from '../Text';
+import StyledCheckbox from '../StyledCheckbox';
+import Link from '../Link';
+
+import { updateSettingsMutation } from './mutations';
+
+const messages = defineMessages({
+  title: {
+    id: 'updates',
+    defaultMessage: 'Updates',
+  },
+  mainDescription: {
+    id: 'EditCollective.Updates.description',
+    defaultMessage:
+      'Updates is a way to keep your community posted on your progress. Once enabled, a new "Updates" section will be added to your profile page and a dedicated page will be created on {updatesLink}.',
```

Comment:
**[:ice_cream: Details & personal preferences]** For me the most important thing about this feature is that it allows you to send a real **newsletter** (it's not just a blog post), I think we should somehow mention it.

```suggestion
      'Updates is a way to keep your community posted on your progress. They are sent by email to all you contributors and followers. Once enabled, a new "Updates" section will be added to your profile page and a dedicated page will be created on {updatesLink}.',
```

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
Agreed. Good suggestion. Let's merge that :-)
Thank you for helping in this PR, really appreciated!